modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+- Pressing Shift + Enter will now step out of `blockquote` elements to match `pre` element behavior #TINY-6559gp
```

Comment:
This is no longer the case it needs to be updated to include the changes to the option and that the default is blockquote. Also this like has `gp` at the end of the text not sure what that is? Maybe it needs to be two log items since we are altering the option and also changing the default behavior to split blockquotes not sure what make sense.